### ISSUE TITLE
Add a new button to archival object pages to provide users with additional information about requesting

### ISSUE THREAD

**Issue by erussey** [emory-libraries/aspace#140]
Create a button that points to a help page to help patrons with requesting.  This button should open a new tab, which is what is the default behavior of the other buttons.  These buttons appear on archival object and resource object display pages.  

- [x] Expand icons to include a help button with text of 'request help' . I'm open as to the icon image, but maybe something with a question mark?

- [x] Create a new page with the URL: https://archives-test.libraries.emory.edu/help-requesting . The page can have default text right now and I will supply soon in comments below.  It should have the same header, footer, and styling of the existing help page: https://archives-test.libraries.emory.edu/help

- [x] Add the new icon to appear by default on all resource and archival object pages. New button behavior is to appear to the right of the other ASpace icons (which appear variably in these pages), but to the left of the Aeon plug-in button/status box.  Clicking on the button will open a new tab that links to the new page above.

Examples:

https://archives-test.libraries.emory.edu/repositories/7/resources/4339
![Screenshot 2023-04-28 at 9.12.23 AM.png](https://images.zenhubusercontent.com/62ff941ccf40d827200a9edf/8cd62791-2084-47d6-85fc-c7faf371e19b)

https://archives-test.libraries.emory.edu/repositories/7/archival_objects/499592
![Screenshot 2023-04-28 at 9.13.44 AM.png](https://images.zenhubusercontent.com/62ff941ccf40d827200a9edf/e6a1e9f4-1b83-46b1-9955-cc0b9e90a0c4)

https://archives-test.libraries.emory.edu/repositories/7/archival_objects/504726
![Screenshot 2023-04-28 at 9.15.37 AM.png](https://images.zenhubusercontent.com/62ff941ccf40d827200a9edf/7fc58299-6fae-4d7c-9fac-531cbcbbaf74)

**Comment by maxdmayhew** [comment]
Since the "help requesting" page will still be within the archives website, best practice is to keep from opening a new tab. Here is what I have so far. 

![Screenshot 2023-04-28 at 6.53.53 PM.png](https://images.zenhubusercontent.com/633dacae52c9e27764e5849b/0cdd9e57-ad44-43b5-bf8d-d0e2446a3a4c)

The layout might need to be tweaked (because of the "No Access" & "No requestable containers" messages) but we can reassess on the next deploy.

**Comment by bwatson78** [comment]
Backend's responsibility has been fulfilled by https://github.com/emory-libraries/aspace_static_plugins/pull/46, which has been approved and merged into `main`.

**Comment by maxdmayhew** [comment]
![Screenshot 2023-05-01 at 12.53.30 PM.png](https://images.zenhubusercontent.com/633dacae52c9e27764e5849b/2c1b3585-e2c3-41f6-88cd-ae10e5171000)

button now links to help-requesting page with placeholder text.

submitting PR

**Comment by erussey** [comment]
@maxdmayhew : A couple of text changes:

- Please change the button label to "How to request"

- Please change the page header that button links to to "How to Request Materials"

Page link is still fine, I think.

**Comment by erussey** [comment]
@maxdmayhew : What is the update on this story?  It looks like the final task is not done yet?  I realize that the help page does not have text yet, which the archivists are still working on.  Could we release the button/dummy page to test once the last task is done and put a story in the next sprint to add the text?  

Please make sure that the help page opens in a new tab.  I know that isn't best practice, but it is what we are doing for the header and will ensure that patrons visiting the help page do not have to redo their search.

**Comment by maxdmayhew** [comment]
PR Submitted. I made the link open in a new tab but links in the header open in new tabs because they link to external sites.

We want to keep things consistent from a UX perspective and this behavior can be disorienting for users.

**Comment by erussey** [comment]
The button structure here is not ideal.  @maxdmayhew : I'm not sure if anything can be done about this, but the buttons get jumbled with longer titles:

![Screenshot 2023-05-10 at 10.15.28 AM.png](https://images.zenhubusercontent.com/62ff941ccf40d827200a9edf/b1e7ded8-d779-43ab-85b7-6488dd0825a4)

**Comment by maxdmayhew** [comment]
So this is because of the "No requestable containers" alert, not the title. However, it's still an issue. 


![Screenshot 2023-05-10 at 11.13.50 AM.png](https://images.zenhubusercontent.com/633dacae52c9e27764e5849b/264aed0e-d052-4515-a748-148bd3272bb1)

I think I can get the "no requestable containers" alert to show up last like I have here. And push everything to the right side. What do you think of this @erussey?

**Comment by maxdmayhew** [comment]
submitting PR, this is sort of a test since the "no requestable containers" alert doesn't show up on my local instance.

**Comment by maxdmayhew** [comment]
https://github.com/emory-libraries/aspace_static_plugins/pull/49
